### ISSUE TITLE
Unix-ify the config and rank files.

### ISSUE THREAD

**Issue by fyrefiend** [msokalski/ascii-patrol#25]
First off, please forgive me as I am *not* a programmer or I'd try to see if I could do this myself.

On *nix systems config files like asciipat.cfg, asciipat.rec, asciipat.rnk are normally hidden by appending a '.' to the beginning of the file name or putting them in a hidden directory that starts with a '.'. So, the config files would be .asciipat.cfg, etc or in a directory called something like .asciipat/, or .config/asciipat.

It would be wonderful if when compiled for *nix systems asciipat followed this convention.

**Comment by msokalski** [comment]
Yes you are right. I'll change it in my spare time. It will also require a configuration detection and migration mechanism to ensure that users transitioning from the current version to the next version do not suffer.
